app/public/js/home.js

```diff
@@ -227,4 +228,39 @@ function bindCategoryLinksToKeywordInput()
 	updateCategoryLinksOffKeywords();
 }
 
-$(document).ready(bindCategoryLinksToKeywordInput);
\ No newline at end of file
+$(document).ready(bindCategoryLinksToKeywordInput);
+
+
+function blinkMap(map1)
+{
+	$(map1).fadeOut(200, function(){ $(map1).fadeIn(200); });
+	//fadeOut(map1,0.5)
```

Comment:
Clean up the commented code and unused functions.  We don't want code we don't currently use in our code base.

Reply:
Got it, fixed that